app/src/main/java/net/squanchy/schedule/view/ScheduleView.java

```diff
@@ -32,11 +36,31 @@ protected void onFinishInflate() {
         ViewPager viewPager = (ViewPager) findViewById(R.id.viewpager);
         TabLayout tabLayout = (TabLayout) findViewById(R.id.tabstrip);
         tabLayout.setupWithViewPager(viewPager);
+        hackToApplyTypefaces(tabLayout);
 
         viewPagerAdapter = new ScheduleViewPagerAdapter(getContext());
         viewPager.setAdapter(viewPagerAdapter);
     }
 
+    private void hackToApplyTypefaces(TabLayout tabLayout) {
+        // Unfortunately doing this the sensible way (in ScheduleViewPagerAdapter.getPageTitle())
+        // results in a bunch of other views on screen to stop drawing their text, for reasons only
+        // known to the Gods of Kobol. We can't theme things in the TabLayout either, as the
+        // TextAppearance is applied _after_ inflating the tab views, which means Calligraphy can't
+        // intercept that either. Sad panda.
+        tabLayout.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
+            Typeface typeface = TypefaceUtils.load(getContext().getAssets(), "fonts/LeagueSpartan-Bold.otf");
+            int tabCount = tabLayout.getTabCount();
+            for (int i = 0; i < tabCount; i++) {
+                TabLayout.Tab tab = tabLayout.getTabAt(i);
+                if (tab == null) {
```

Comment:
Not sure but it's annotated as `@Nullable` and there's a ternary that can return null inside, so I went for it